docs/csharp/deconstruct.md

```diff
@@ -45,11 +45,17 @@ There are two ways to deconstruct a tuple:
 
     This is cumbersome and is not recommended.
 
+- Lastly, you may deconstruct the tuple into variables that have already been declared.
+
+    [!code-csharp[Deconstruction-Declared](../../samples/snippets/csharp/programming-guide/deconstructing-tuples/deconstruct-tuple5.cs#1)]
+
 Note that you cannot specify a specific type outside the parentheses even if every field in the tuple has the
 same type. This generates compiler error CS8136, "Deconstruction 'var (...)' form disallows a specific type for 'var'.".
 
 Note that you must also assign each element of the tuple to a variable. If you omit any elements, the compiler generates error CS8132, "Cannot deconstruct a tuple of 'x' elements into 'y' variables."
 
+Note that you cannot mix declarations and assignments to existing variables on the left-hand side of a deconstruction. If you try to deconstruct a tuple into newly declared variables as well as variables that have already been declared, the compiler generates error CS8184, "A deconstruction cannot mix declarations and expressions on the left-hand-side."
+
```

Comment:
I'd re-word the second sentence as follows:

> The compiler generates error CS8184, "a deconstruction cannot mix declarations and expressions on the left-hand-side." when the members include newly declared and existing variables.

Reply:
Thanks for the feedback! That sounds much less awkward than what I had written.